src/Database/DBStructure.php

```diff
@@ -517,6 +541,10 @@ private static function FieldCommand($parameters, $create = true) {
 			$fieldstruct .= " ".$parameters["extra"];
 		}
 
+		if (!is_null($parameters["comment"])) {
```

Comment:
Isn't this directly contradicting with this: https://github.com/friendica/friendica/pull/4229/files#diff-43062db8a7c49dc0c0ff03f5667b5badR320 ?

Reply:
The linked function is called from different places - not only when a table definition changed but also when the table was new.

Reply:
Ok, just checking.